### PR TITLE
fix #79956 guardyoureyes.com

### LINKED ISSUE #79956: www.guardyoureyes.com
### Issue URL (Annoyance)
 [https://www.guardyoureyes.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.guardyoureyes.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (2880), <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances

### PR DESCRIPTION
**Pull request by wout-codes**
#79956

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -98,6 +98,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+guardyoureyes.com#$##specialModal { display: none !important; }
```
Comment (Sergey-Lyapin) [inline AnnoyancesFilter/sections/popups.txt:101]: what about
`||guardyoureyes.com/templates/mentalpress/js/subscription.js`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt:101]: Do you check the whole site from top to bottom or what?

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt]: Is something wrong? This site is unavailable at the moment

Comment (wout-codes) [reply AnnoyancesFilter/sections/popups.txt]: Nope. You find things which I do not find, you reproduce things which I can't reproduce.

**Pull request merged**